test/specs/calculator.add.multiply.spec.js

```diff
@@ -0,0 +1,158 @@
+    expect(() => calculator.add(undefined)()).to.throw('Unable to calculate, [undefined] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'number\' and \'increment\' are not defined(undefined) an error is thrown', () => {
+    expect(() => calculator.add(undefined, undefined)()).to.throw('Unable to calculate, [undefined] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'number\' is not defined(undefined) an error is thrown', () => {
+    expect(() => calculator.add(undefined, 5)()).to.throw('Unable to calculate, [undefined] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'increment\' is not defined(undefined) an error is thrown', () => {
+    expect(() => calculator.add(68, undefined)()).to.throw('Unable to calculate, [undefined] is not a number');
+  });
+});
+
+// eslint-disable-next-line max-len
+describe('It should test Calculator class method \'multiply\' positive scenarios', () => {
+  let multiplier;
+  beforeEach(() => {
+    multiplier = new Calculator();
+  });
+  afterEach(() => {
+    multiplier = null;
+  });
+  it('It shout test the calculation of the natural positive numbers', () => {
+    expect(multiplier.multiply(865, 45)).to.be.equal(38925);
+  });
+  it('It should test the calculation with the negative number', () => {
+    expect(multiplier.multiply(-356, 125)).to.be.equal(-44500);
+  });
+  it('It should test the calculation with whole numbers', () => {
+    expect(multiplier.multiply(548, 0)).to.be.equal(0);
+  });
+  it('It should test the calculation with fractions', () => {
+    expect(multiplier.multiply(3 / 8, 4 / 8)).to.be.equal(0.1875);
+  });
+  it('It should test the calculation with the rational numbers', () => {
+    // eslint-disable-next-line max-len
+    expect(multiplier.multiply(-245.969, 138.354)).to.be.equal(-34030.795026);
+  });
+});
+
+// eslint-disable-next-line max-len
+describe('It should test Calculator class method \'multiply\' negative scenarios', () => {
+  let multiplier;
+  beforeEach(() => {
+    multiplier = new Calculator();
+  });
+  afterEach(() => {
+    multiplier = null;
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'number\' is not defined(null) an error is thrown', () => {
+    expect(() => multiplier.multiply(68, null)()).to.throw('Unable to multiply, [null] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'increment\' is not defined(null) an error is thrown', () => {
+    expect(() => multiplier.multiply(68, null)()).to.throw('Unable to multiply, [null] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'number\' and \'multiplier\' are not defined(null) an error is thrown', () => {
+    expect(() => multiplier.multiply(null, null)()).to.throw('Unable to multiply, [null] is not a number');
```

Comment:
It's a bad practice to test a scenario with 2 invalid arguments. Please correct.

Reply:
Why? Due to the fact that our methods can receive any amount of any values I don't think it's an invalid check.

Reply:
In this case, can not be sure of the exact reason for your test being failed.